pkg/term/proxy.go

```diff
@@ -51,6 +51,10 @@ func (r *escapeProxy) Read(buf []byte) (int, error) {
 		return nr, err
 	}
 
+	if len(r.escapeKeys) == 0 {
+		return nr, nil
```

Comment:
This is never reached, in order to raech this you will need to add a test case that actually has no escapeKeys and more only one key, cases with more than one key are already caught here: https://github.com/moby/moby/blob/111457538761414254a8c1a3eb2ac9654b36aff8/pkg/term/proxy.go#L47-L52